docker-compose.yml

```diff
@@ -93,6 +93,21 @@ services:
       - ./api/docker/caddy/Caddyfile:/etc/caddy/Caddyfile:ro
       - ./api/public:/srv/api/public:ro
 
+  http-cache:
+    image: varnish:7.4.2-alpine
+    container_name: 'ecamp3-http-cache'
+    depends_on:
+      - caddy
+    volumes:
+      - ./api/docker/varnish/vcl/:/etc/varnish/:ro
+    tmpfs:
+      - /var/lib/varnish/varnishd:exec
+    ports:
+      - target: 8080
+        published: 3004
+        protocol: tcp
+    command: -a :8080 -a :8081 -p http_max_hdr=96
```

Comment:
Yes, I should adjust this to how I've set up Kubernetes.

The way the varnish docker entrypoint is programmed, it would always fall back to port 80 if VARNISH_HTTP_PORT is not provided. So above would actually make varnish listening on 80, 8080 and 8081.

Reply:
Fixed with https://github.com/ecamp/ecamp3/pull/3610/commits/60cc980e6db98e950ef168cbe64be7f60fea64d9